### ISSUE TITLE
have some trouble

### ISSUE THREAD

**Issue by yelanlan** [mczhuge/ICON#9]
Hi, I trained with the following config and I did not change the code
python main/train.py --model 'ICON-S' --dataset '/data2/lanjun/dataset/sod_dataset/datasets/DUTS/Train' --lr 2e-3 --decay 2e-4 --momen 0.9 --batchsize 6 --loss 'CPR' --savepath 'checkpoint/finetune/ICON-S/' --valid True

but I get the following result，it doesn't match the result in the paper. Did I do something wrong? need help
Method:ICON-valid-59,Dataset:ECSSD||Smeasure:0.927; meanEm:0.954; wFmeasure:0.911; MAE:0.032; fnr:0.065||adpEm:0.955; meanEm:0.954; maxEm:0.963; adpFm:0.918; meanFm:0.916; maxFm:0.941
Method:ICON-valid-59,Dataset:PASCAL-S||Smeasure:0.875; meanEm:0.912; wFmeasure:0.834; MAE:0.054; fnr:0.115||adpEm:0.908; meanEm:0.912; maxEm:0.921; adpFm:0.843; meanFm:0.849; maxFm:0.872
Method:ICON-valid-60,Dataset:ECSSD||Smeasure:0.926; meanEm:0.953; wFmeasure:0.911; MAE:0.032; fnr:0.069||adpEm:0.955; meanEm:0.953; maxEm:0.962; adpFm:0.919; meanFm:0.917; maxFm:0.94
Method:ICON-valid-60,Dataset:PASCAL-S||Smeasure:0.874; meanEm:0.91; wFmeasure:0.833; MAE:0.055; fnr:0.122||adpEm:0.907; meanEm:0.91; maxEm:0.919; adpFm:0.843; meanFm:0.849; maxFm:0.87

**Comment by mczhuge** [comment]
The results you obtained are indeed one point lower than the paper, so if it is convenient for you, please retrain a version (adjust the learning rate) and provide me with additional feedback. Actually, in my experiments, the ICON-S, ICON-P have very stable performance. So I will try it in my side.

**Comment by yelanlan** [comment]
I use the default config as follow, what learning rate should I set?
python main/train.py --model 'ICON-S' --dataset '/data2/lanjun/dataset/sod_dataset/datasets/DUTS/Train' --lr 2e-3 --decay 2e-4 --momen 0.9 --batchsize 6 --loss 'CPR' --savepath 'checkpoint/finetune/ICON-S/' --valid True

**Comment by yelanlan** [comment]
Sorry, I did something wrong, and I get the same result with you. Thank you

**Comment by mczhuge** [comment]
@yelanlan  Thanks for feedback. This is my WeChat: tjpxiaoming.
